.github/workflows/cicd.yaml

```diff
@@ -136,19 +133,6 @@ jobs:
       - name: Install playwright browsers
         run: pnpm playwright install --with-deps
 
-      - name: Setup screenshots cache
-        id: screenshot-cache
-        uses: actions/cache@v4
-        with:
-          key: ${{ runner.os }}-build-${{ github.ref }}
-          path: src/**/*-snapshots/*
-
-      - name: Initialize screenshots
-        if:
-          ${{steps.screenshot-cache.outputs.cache-hit != 'true' || inputs.update-snapshots ==
-          'true'}}
-        run: pnpm e2e --update-snapshots
-
       - name: Run tests
         run: pnpm e2e
```

Comment:
You are now running a different script, since github actions is already running in a Linux docker container with the required tools installed there is really no need to run another container-in-container. you can just execute the 'internal' script.

Reply:
But that will be a different image/OS from the one we run locally, right? By using the same docker image locally and in CI, we should get consistent screenshots

Reply:
I think they are already matching, at least I get no differences whether I run the images in github actions vs locally vs through docker compose. 

Anyway, if you do want to use the docker compose action then we can remove the previous steps in the e2e job that installs dependencies, pnpm, builds storybook etc because the docker-compose.yml already defines that as the command to run.

Reply:
I think it is the same if running on a linux machine. On the mac, the screenshots are generated with a `-darwin` suffix

Good catch on the unnecessary steps, removed